src/commands/doctor.rs

```diff
@@ -0,0 +1,96 @@
+use std::{fmt::Debug, path::PathBuf};
+
+use anyhow::Result;
+use clap::Parser;
+use dialoguer::{console::Emoji, Select};
+use futures::FutureExt;
+use spin_doctor::Diagnosis;
+
+#[derive(Parser, Debug)]
+#[clap(hide = true, about = "Run the Spin Doctor")]
+pub struct DoctorCommand {
+    #[clap(short = 'f', long, default_value = "spin.toml")]
+    file: PathBuf,
+}
+
+impl DoctorCommand {
+    pub async fn run(self) -> Result<()> {
+        eprintln!("{icon}The Spin Doctor is in.", icon = Emoji("📟 ", ""));
```

Comment:
To throw my opinion in the ring: I quite like emoji because I do find that it helps guide my eye, but more than anything I would prefer *color* output. I find spin output generally very hard to read because it all comes out looking like a wall of text.